### PR TITLE
Implement PLE0206

### PR DESCRIPTION
**Pull request by harupy**
#970

### CODE REVIEW

File: src/pylint/plugins.rs [hunk 1]
```diff
@@ -1,10 +1,37 @@
...
+pub fn property_with_parameters(
+    checker: &mut Checker,
+    stmt: &Stmt,
+    decorator_list: &[Expr],
+    args: &Arguments,
+) {
+    if decorator_list.iter().any(|d| match &d.node {
+        ExprKind::Name { id, .. } if id == "property" => true,
+        _ => false,
+    }) {
```
Comment (charliermarsh) [inline src/pylint/plugins.rs:18]: Maybe also check `checker.is_builtin(property)`?

**Comment by charliermarsh** [comment]
Rock! Nice work.

**Comment by charliermarsh** [comment]
Oh wait, I think this should be `PLR0206`? Since it's a "refactor" rule?

**Comment by harupy** [comment]
@charliermarsh I misunderstood the naming rule. Yes, it should be `PLR0206`.

**Comment by charliermarsh** [comment]
@harupy - Sorry, I wasn't super clear. What do you think of that convention? I wanted to preserve the `R0206` part to match Pylint.

**Comment by harupy** [comment]
That convention sounds good to me. If we use `PLE`, `W1301` and `E1301` would conflict.

**Comment by charliermarsh** [comment]
Oh hah, true!

**Comment by charliermarsh** [comment]
I'll rename this check and push it straight to `main`.

**Comment by charliermarsh** [comment]
Oh nm! Thank you!

**Pull request merged**